modules/addStaticOnLocalStorage.js

```diff
@@ -0,0 +1,117 @@
+import { usersSettings } from './usersSettings.js'
+
+// Добавляет статистику по загрузкам в localStorage
+function addStaticOnLocalStorage() {
+    let elem = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload")
+
+    let promise = new Promise((resolve, reject) => {
+        let observer = new MutationObserver((mutationRecords) => {
+            if (mutationRecords.length > 1) {
+                let saveScanStat = document.querySelector("#b-container > div.b-popup-wrapper.js-popup-wrapper.js-popup-wrapper--upload > div.b-popup.b-popup--blue.b-popup--upload.js-popup--upload.b-popup--edit.js-popup--edit > form > div.b-popup-button.js-upload-submit"); // кнопка по которой срабатывает скрипт
```

Comment:
please try mot to shorten the names, not to be afraid of using long names. They are descriptive and takes less time to understand what's going on :)